.github/workflows/snap.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       fail-fast: false
       matrix:
         architecture: ${{ fromJSON(needs.arches.outputs.arches) }}
-        base: ["22", "24"]
```

Comment:
If you drop this, you need to stop using it:
```
        publish-channel: ${{ matrix.track }}/edge/mir-pr${{ github.event.number }}
```

Reply:
Yeah yeah. I know. Just looking at fixing the spread runs now, since [`lsb-invalid-mta`](https://packages.ubuntu.com/jammy/lsb-invalid-mta) is gone in noble.